docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -15,7 +15,7 @@ The media files are updated by an application (by using the classes and methods
 You can only use the Tizen.Content.MediaContent classes to manage files located in specific paths.
 
 
-> **Note**   
+> **Note**
```

Comment:
```suggestion
> [!NOTE]
```
Formatting of Alerts in Tizen Docs have been updated.
Reference: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning